prebuilt-ui/basic-embed/components/Header.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+export const Header = () => (
+  <header className="header">
+    <div className="row">
+      <img src="daily-logo.svg" alt="Daily" className="logo" />
+      <div className="capsule">Prebuilt demo</div>
+    </div>
+    <div className="row">
+      <div className="capsule">
+        <a href="https://docs.daily.co/docs/">API docs</a>
+      </div>
+      <span className="divider"></span>
+      <a href="https://github.com/daily-demos/examples/tree/main/prebuilt-ui/basic-embed">
```

Comment:
can you add `target="_blank" rel="noopener noreferrer"` to any external links please?

Reply:
Done, thanks!